python/lsst_versions/_versions.py

```diff
@@ -384,28 +384,50 @@ def _process_version_writing(
     write_to: Optional[str] = None
     written = None
     if write_version:
-        write_to = _find_version_path(dir)
+        write_to = _find_version_path(dirname)
         if write_to is None:
             return "<unknown>", written
 
     # Find the version of HEAD and current directory.
```

Comment:
Please change this chunk of code to call the new get_lsst_version. I don't like code duplication.